tools/nnpackage_tool/model2nnpkg/README.md

```diff
@@ -7,17 +7,47 @@ It takes `modelfile` as input and generates `nnpackage`.
 ## Usage
 
 ```
-Usage: model2nnpkg.sh [options] modelfile
-Convert modelfile to nnpackage.
+Usage: model2nnpkg.sh [options]
+Convert modelfile (tflite, circle or tvn) to nnpackage.
 
 Options:
     -h   show this help
     -o   set nnpackage output directory (default=.)
-    -p   set nnpackage output name (default=[modelfile name])
+    -p   set nnpackage output name (default=[1st modelfile name])
+    -c   provide configuration files
+    -m   provide model files
 
 Examples:
-    model2nnpkg.sh add.tflite                  => create nnpackage 'add' in ./
-    model2nnpkg.sh -o out add.tflite           => create nnpackage 'add' in out/
-    model2nnpkg.sh -o out -p addpkg add.tflite => create nnpackage 'addpkg' in out/
+    model2nnpkg.sh -m add.tflite                                              => create nnpackage 'add' in ./
+    model2nnpkg.sh -o out -m add.tflite                                       => create nnpackage 'add' in out/
+    model2nnpkg.sh -o out -p addpkg -m add.tflite                             => create nnpackage 'addpkg' in out/
+    model2nnpkg.sh -c add.cfg -m add.tflite                                   => create nnpackage 'add' with add.cfg
+    model2nnpkg.py -o out -p addpkg -m a1.tflite a2.tflite => create nnpackage "addpkg" with models a1.tflite and a2.tflite in out/
 
 ```
+
+## Usage of python script
```

Comment:
@ragmani Please find https://github.com/Samsung/ONE/pull/10221#issuecomment-1360528234. If we choose to make new python compatible to bash script, I would like to hide the implementation. Just introduce `model2nnpkg` and keep just one interface in `README.md`. I would like to hear other's opinion. 

cc @Samsung/one_onert, @hseok-oh, @chunseoklee